### PR TITLE
Implementation of fine-grained provenance display following designs, as well as display of hospital accessibility features panel. Updates to readme and factoring.

### PR DESCRIPTION
**Pull request by amb26**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,9 +21,20 @@ To manually compile Scss files into CSS files, run
 
     npm run build:scss
 
-## Build & Run
+## Testing
 
 To quickly see the interface in action, load `index.html` from this directory in your browser from a local static HTTP server.
+This index sources live data from GitHub from the [covid-assessment-centres](https://github.com/inclusive-design/covid-assessment-centres) repository.
```
Comment (cindyli) [inline README.md]: "from GitHub from the ..." -> "from the Github ..."

File: src/css/scss/fluid-covid-map-viz.scss [hunk 2]
```diff
@@ -518,6 +518,47 @@ button {
...
+.fl-mapviz-hospital-prov {
+    flex-direction: row;
+    display: flex;
+    align-items: center;
+    background: #E7E7E7;
+    height: 4rem;
+}
+
+
+.fl-mapviz-hospital-features-prov {
```
Comment (cindyli) [inline src/css/scss/fluid-covid-map-viz.scss]: Both `.fl-mapviz-hospital-features-prov` and `.fl-mapviz-hospital-description-prov` are extended from `.fl-mapviz-hospital-prov` without any change. Moreover, `.fl-mapviz-hospital-prov` is a css definition that is not used in the markup. Would it be easier to understand if the css for all three selectors is defined as:

```
.fl-mapviz-hospital-features-prov,
.fl-mapviz-hospital-description-prov {
    {a copy of line 545-549}
}
```

**Comment by cindyli** [comment]
The hospital panel on the mobile view is completely taken by provenance bars and the panel is not scrollable.
mobile

@uttaraghodke, can you add a mobile design for the provenance display? Thanks.

**Comment by cindyli** [comment]
@amb26 I issued https://github.com/amb26/covid-data-monitor/pull/3 on your branch. It solves the scrollability issue with the hospital panel on the mobile view. I also explained a couple of other things on its PR description.

**Comment by amb26** [comment]
Thanks for the excellent fixes, @cindyli, ready for another look

**Pull request merged**